paasta_tools/setup_prometheus_adapter_config.py

```diff
@@ -248,14 +248,15 @@ def should_create_active_requests_scaling_rule(
 
 def create_instance_active_requests_scaling_rule(
     service: str,
-    instance: str,
-    autoscaling_config: AutoscalingParamsDict,
+    instance_config: KubernetesDeploymentConfig,
     paasta_cluster: str,
-    namespace: str = "paasta",
```

Comment:
@nemacysts thanks for the review. but as you can see here, the default value is actually set the method level but not at a level before. Because I have removed the default value, I was doing the 

```
instance_config.get_namespace() or "paasta"
```